pkg/volume/plugins.go

```diff
@@ -384,6 +391,36 @@ func (spec *Spec) Name() string {
 	}
 }
 
+// IsControllerExpandable returns true for volume types that can be expanded by the controller
+// and false for those that need to be run on the node. Flex volume is the only one in the
+// latter category since its typically not installed on the controller/master
+func (spec *Spec) IsControllerExpandable() bool {
+	switch {
+	case spec.Volume != nil:
+		return spec.Volume.FlexVolume == nil
+	case spec.PersistentVolume != nil:
+		return spec.PersistentVolume.Spec.FlexVolume == nil
+	default:
+		return true
+
+	}
+}
+
+// NonControllerExpandablePluginName creates and returns a name for the plugin
+// this is used in context on the controller where the plugin lookup fails
+// as volume expansion on controller isn't supported, but a plugin name is
+// required
+func (spec *Spec) NonControllerExpandablePluginName() string {
+	switch {
+	case spec.Volume != nil:
```

Comment:
Add check for flexvolume, this would return true for all plugins?

Reply:
Yes, missed that. Done.